src/components/ComposerAttachments.vue

```diff
@@ -134,11 +134,11 @@ export default {
 			return done
 		},
 		onAddCloudAttachment() {
-			const picker = getFilePickerBuilder(t('mail', 'Choose a file to add as attachment')).build()
+			const picker = getFilePickerBuilder(t('mail', 'Choose a file to add as attachment')).setMultiSelect(true).build()
 
 			return picker
 				.pick(t('mail', 'Choose a file to add as attachment'))
-				.then((path) => this.emitNewAttachment(this.fileNameToAttachment(path)))
+				.then((path) => this.emitNewAttachments(path.map(p => this.fileNameToAttachment(p))))
```

Comment:
```suggestion
				.then((paths) => this.emitNewAttachments(paths.map(this.fileNameToAttachment)))
```

A bit shorter and the correct plural naming makes the code easier to comprehend for the next person touching it :wink: